packages/core/src/html/linkProcessor.ts

```diff
@@ -237,6 +249,7 @@ function collectSource(node, rootPath, baseUrl, pageSources) {
       return;
     }
 
+    if (!node.attribs) return;
```

Comment:
Minor nit: would be better to combine this early return check with the previous one